packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md

```diff
@@ -0,0 +1,403 @@
+    "token_type": "Bearer",
+    "expires_in": 3600,
+    "access_token": "eyJra....3pcxrrhSAw",
+    "scope": "offline_access openid device_sso",
+    "refresh_token": "FOxRzDPAGtOapDzap-rNBOSWznClz3p-zypbZ157W6c",
+    "id_token": "eyJraWQiOi....VQT8GGmg",
+    "device_secret": "+oUXe6pnhkDOTTjR5ZGFQoZGVBrQPiDsUWIk27ioyhM="
+}
+```
+
+> **Note:** Store the `device_secret` in a secure location on the device, such as the iOS keychain. Ensure that only authorized applications have access to the device secret.
+
+### Exchange existing tokens from client 1 for new tokens for client 2
+
+When you make a token exchange request, the response returns the tokens that you need to use for client 2.
+
+**Example request**
+
+```bash
+  curl --location --request POST \
+  --url https://${yourOktaDomain}/oauth2/{authorizationServerId}/v1/token \
+  --header 'Accept: application/json' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'client_id={client #2 id}' \
+  --data-urlencode 'grant_type=urn:ietf:params:oauth:grant-type:token-exchange' \
+  --data-urlencode 'actor_token={device_secret}' \
+  --data-urlencode 'actor_token_type=urn:x-oath:params:oauth:token-type:device-secret' \
+  --data-urlencode 'subject_token={id_token}' \
+  --data-urlencode 'subject_token_type=urn:ietf:params:oauth:token-type:id_token' \
+  --data-urlencode 'scope=openid offline_access' \
+  --data-urlencode 'audience={audience}'
+```
+
+Note the parameters that are being passed:
+
+* `client_id`: Identifies the new client (for example, client #2) and matches the Client ID of the OAuth 2.0 application that you created.
+* `grant_type`: Identifies the mechanism that Okta uses to authorize the creation of the tokens. Value: `urn:ietf:params:oauth:grant-type:token-exchange`
+* `actor_token`: Identifies the `device_secret` that was returned in response to the authorization code request.
+* `actor_token_type`: Identifies the type of token in the `actor_token` parameter.
+* `subject_token`: Identifies the `id_token` that was returned in response to the authorization code request.
+* `subject_token_type`: Identifies the type of token in the `subject_token` parameter. Value: `urn:ietf:params:oauth:token-type:id_token`
+* `scope`: The scopes that you want to be included in the access token.
+* `audience`: Identifies the audience of the authorization server. Locate the value for the `audience` parameter in the Admin Console. From the left navigation pane, select **Security** > **API**. On the **Authorization Server** tab, select the pencil icon for the authorization server that you need the `audience` value for.
+
+If the request is successful, the response returned includes the following tokens: `id_token`, `refresh_token`, and `access token`.
+
+> **Note:** You can pass an expired ID token as part of the token exchange grant as long as the `device_secret` (`sid`) that the `id_token` is associated with is still valid.
+
+**Response example**
+
+```JSON
+{
```

Comment:
Where are the `ID Tokens` and `Refresh tokens`. Especially since offline_access scope was requested in this token exchange call, the response must include refresh tokens.

Example:
{
    "token_type": "Bearer",
    "expires_in": 3600,
    "access_token": "eyJraWQiOiJRd0VDRGs5MFlSenZPNWRqeFJKaGdPcXRTcVFxWExMdGJIYXl0bDlSZS1jIiwiYWxnIjoiUlMyNTYifQ.eyJ2ZXIiOjEsImp0aSI6IkFULk5mS0RKYVdkWFM4VEtLVjRWNnRNLVgxazlMYnp3YzV0SkRsd3diVkNxU28ub2FyMTZicTc3NWdEUExPN1EwaDciLCJpc3MiOiJodHRwczovL2xvZ2luLmhhaWt1dGhlZG9nLmNvbS9vYXV0aDIvYXVzejVlZnB6alhIczNneUEwaDciLCJhdWQiOiJodHRwczovL2FwaS5hdXRvZGVzay5jb20iLCJpYXQiOjE2Mjc5MzkyNjEsImV4cCI6MTYyNzk0Mjg2MSwiY2lkIjoiMG9hdXdjcjd0cWg4OXI2YzYwaDciLCJ1aWQiOiIwMHVqejF5M3hjRTdrUVp3MzBoNyIsInNjcCI6WyJvZmZsaW5lX2FjY2VzcyIsIm9wZW5pZCJdLCJzdWIiOiJ2ZW5rYXQudmlzd2FuYXRoYW5Ab2t0YS5jb20ifQ.jk8RpzBbPvyyqc2hAeHrwAWok2-WjDSSbyVItXbtutpX9Nf3z6E7mt7QEnqZ4Jbrm1GYuJUcq6EZaIBzxt_UAKsr_JlHeZ99swJQSpFTetBydWAIijGkA68ofUUk7WCM7xXRuhilFkJHZ1fjl-xR9ELJoe2izO0UMJGZYVpF5Cd0KLfIvQpYJljgd5yPmffg2GaPHxmivyYxWQyQuvfNYkJGT_LRsa2FG7LC6PRmZDpJJrox6aGB7pDM1UrRw4QY550C7U5y2BlgrVm3f8bSTHCmEw6txxGY3Rn8HpWErlp_GO7xt6Iil-BUEcDePAv3LfOcB8t1f22XT9syM8cRkg",
    "scope": "offline_access openid",
    "refresh_token": "BC3Xgk9TGyDx0SIcOCBh7dJndxDYLUh7NhOYGFwDbAY",
    "id_token": "eyJraWQiOiJRd0VDRGs5MFlSenZPNWRqeFJKaGdPcXRTcVFxWExMdGJIYXl0bDlSZS1jIiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiIwMHVqejF5M3hjRTdrUVp3MzBoNyIsInZlciI6MSwiaXNzIjoiaHR0cHM6Ly9sb2dpbi5oYWlrdXRoZWRvZy5jb20vb2F1dGgyL2F1c3o1ZWZwempYSHMzZ3lBMGg3IiwiYXVkIjoiMG9hdXdjcjd0cWg4OXI2YzYwaDciLCJpYXQiOjE2Mjc5MzkyNjEsImV4cCI6MTYyNzk0Mjg2MSwianRpIjoiSUQuT19qTW1uYUZ2LWozZERWODNVanJuMVdQSFpwcW5jclphN0pGMnR1ZXFjZyIsImFtciI6WyJwd2QiXSwiaWRwIjoiMDBvanoxeTN3YW4xYWl1YzgwaDciLCJzaWQiOiIxMDIwY2kzaE5hd1NodWQ1bWVrMVM0YWh3IiwiYXV0aF90aW1lIjoxNjI3OTM5MjYxLCJhdF9oYXNoIjoiSHBwekFfeE5xR05UYWVzVXhBUzM2QSIsImRzX2hhc2giOiJEel9hSXFPbGpmRXc2UlV1ek9PRDF3In0.IgbzKmc04sA090ZyDnuMZYbaL1k8OWT5YVf2cc289eIu6DyDKZWn75DFjGFIx7OzxNOJLpI6qpD1_q7kIqW0KhsohtJ86afOOepqTUQSFz2-c9bBlt4I3GI-8cbj0HC-DRT4E_zidXxTkrtXqaGiww0dt7pCtvl_SVtAh2H_GEvvi2VEwLPpOo6r_u6hC3uv4kPBKh_gNFqNf9E0b-nIoEd26TKeQgyGZMy5j6oS4NWIUFbwBwrqNI5jLLMF6Lm13o5dbreD1dO-TFzlE23GKd0Ra_h_REe_mhSM0cv-I_DQitGzG9yLcxp57Isdya4w8j0DJ3QAMzEtzvAqS39KOQ",
    "issued_token_type": "urn:ietf:params:oauth:token-type:access_token"
}

Reply:
Fixed.

Reply:
Refresh token is missing.

Reply:
Fixed